examples/time_series/nooa_json.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd 
```

Comment:
Can you make this example look like the others? 

There needs to be a whole skeleton for this example to work.